src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -65,6 +65,7 @@ public class HostController : Controller
         [TypeFilter(typeof(EnableDebugModeFilter))]
         public async Task<IActionResult> GetHostStatus([FromServices] IScriptHostManager scriptHostManager, [FromServices] IHostIdProvider hostIdProvider, [FromServices] IServiceProvider serviceProvider = null)
         {
+            // When making changes to HostStatus, ensure that you update the HostStatus class in Antares as well: https://msazure.visualstudio.com/One/_git/AAPT-Antares-Websites?path=/src/Hosting/AdministrationService/Microsoft.Web.Hosting.Administration.Api/WebEntities/Kudu/HostStatus.cs
```

Comment:
I would remove the file path and just mention the class name.

Reply:
Thanks, updated!